appveyor.yml

```diff
@@ -35,11 +35,12 @@ before_deploy:
 
 artifacts:
   - path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe

```

Comment:
And another thing I just saw: it seems that the syntax for env variables in that place is $(envVarName), not %envVarName%. (see the third paragraph on http://www.appveyor.com/docs/packaging-artifacts)